### ISSUE TITLE
renamerOnUpdate (ROU) renames a file as duplicate when updating performers

### ISSUE THREAD

**Issue by thedesi1** [stashapp/CommunityScripts#251]
When updating/adding performers for a scene, the ROU renames the files with trailing `_1` or generically `_n`

**Comment by DogmaDragon** [comment]
That is an intentional feature, as far as I know. 

From the config
```py
# word attached at end if multiple file for same scene [FileRefactor]
duplicate_suffix = ["", "_1", "_2", "_3", "_4", "_5", "_6", "_7", "_8", "_9", "_10"]
```

**Comment by thedesi1** [comment]
Exactly, when multiple files - the bug is if you change a attribute that is not part of the rename i.e. `$studio - $date - $title` in my case performers, the RoU just append `_1` to existing files, when only 1 file exists.

**Comment by thedesi1** [comment]
Also, although I cant prove it, but I am certain that this bug was introduced when this plugin was moved to the new plugin architecture.
